### PR TITLE
Improve consistency

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
Consistency is nice, but why remove the links? It's pretty essential for the landing page that people know from where to get their token. Having this comment is a concise and effective way to present this information.

**Comment by KnorpelSenf** [comment]
Also, instructing people to put their token between the `""` is less error-prone to absolute newcomers than doing something like `"<token>"` or not doing anything at all. It happens fairly often that people use `"<12345:abCDefGH>"` as their token because they don't get it. This style is intentional.

**Comment by KnorpelSenf** [comment]
How about this:
- the landing page is special, so we put the link to `@BotFather` there
- all other pages have _between the `""`_

**Comment by roj1512** [comment]
> the landing page is special, so we put the link to @BotFather there

Yes, I wanted something like that.

**Comment by KnorpelSenf** [comment]
Cool, go ahead then

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]
There are many more places where `new Bot()` is created. We should update them there, too.

File: site/docs/zh/guide/errors.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ grammY 不会以任何方式触及抛出的错误，而是把它包装成一个
 因此，看起来就好像包含错误的中间件运行了 `next`。
 
 ```ts
-const bot = new Bot("");
+const bot = new Bot(""); // <-- put your bot token between the ""
```
Comment (KnorpelSenf) [inline site/docs/zh/guide/errors.md:112]: Wrong language.

**Comment by roj1512** [comment]
No, this is a draft.

**Comment by roj1512** [comment]
*Supposed to be

File: site/docs/es/plugins/transformer-throttler.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ const botToken = process.env.BOT_TOKEN;
 if (!botToken) {
   throw Error("Se requiere BOT_TOKEN");
 }
-const bot = new Bot(botToken);
+const bot = new Bot(""); // <-- put your bot token between the ""
```
Comment (KnorpelSenf) [inline site/docs/es/plugins/transformer-throttler.md:47]: @roj1512 this doesn't make any sense

**Comment by KnorpelSenf** [comment]
We now agreed on the following:

* the landing page is special, so we put the link to `@BotFather` there
* Hosting tutorials are meant for prod, so we show env var examples there
* all other pages have _between the `""`

File: site/docs/guide/middleware.md [hunk 3]
```diff
@@ -253,7 +253,7 @@ Middleware objects are simply a wrapper for middleware functions.
 They are mostly used internally, but can sometimes also help third-party libraries, or be used in advanced use cases, such as with [Composer](https://deno.land/x/grammy/mod.ts?s=Composer):
 
 ```ts
-const bot = new Bot("<token>");
+const bot = new Bot(""); // <-- put your bot token between the ""
```
Comment (KnorpelSenf) [inline site/docs/guide/middleware.md:256]: I don't like this. The topic on this page is completely unrelated to getting your bot up and running. The comment is useless information here, and it distracts from what we actually want to say. What's consistency worth if applied in places it doesn't belong?

File: site/docs/guide/errors.md [hunk 4]
```diff
@@ -107,7 +107,7 @@ In that case, the fenced middleware does not only act as if it had completed suc
 Thus, it looks like the middleware inside the error boundary has called `next`.
 
 ```ts
-const bot = new Bot("");
+const bot = new Bot(""); // <-- put your bot token between the ""
```
Comment (KnorpelSenf) [inline site/docs/guide/errors.md:110]: Same here

File: site/docs/guide/deployment-types.md [hunk 5]
```diff
@@ -276,7 +276,7 @@ If you want to use webhook replies, you can specify the `canUseWebhookReply` opt
 Pass a function that determines whether or not to use webhook reply for the given request, identified by method.
 
 ```ts
-const bot = new Bot(token, {
+const bot = new Bot("", { // <-- put your bot token between the ""
```
Comment (KnorpelSenf) [inline site/docs/guide/deployment-types.md:279]: Same here

File: site/docs/guide/context.md [hunk 6]
```diff
@@ -452,7 +452,7 @@ class MyContext extends Context {
 }
 
 // Pass the constructor of the custom context class as an option.
-const bot = new Bot("<token>", {
+const bot = new Bot("", { // <-- put your bot token between the ""
```
Comment (KnorpelSenf) [inline site/docs/guide/context.md:455]: Same here (and I'm sure there are more places)

**Comment by KnorpelSenf** [comment]
Using an undeclared variable?

**Comment by roj1512** [comment]
Yes

**Comment by KnorpelSenf** [comment]
I think that I like an empty string better than that, it has better pasteability

**Comment by roj1512** [comment]
So just remove the comments except in the main page?

**Comment by KnorpelSenf** [comment]
IMO yes.

**Comment by KnorpelSenf** [comment]
And perhaps leave them in the guide, as well.

**Comment by roj1512** [comment]
Guide?

**Comment by KnorpelSenf** [comment]
Sorry, getting started is what I was thinking of. Not the entire guide.

File: site/docs/advanced/middleware.md [hunk 7]
```diff
@@ -13,7 +13,7 @@ While it is not wrong that you can use middleware in this linear fashion (also i
 Commonly, you see the following pattern.
 
 ```ts
-const bot = new Bot("<token>");
+const bot = new Bot("");
```
Comment (KnorpelSenf) [inline site/docs/advanced/middleware.md:16]:
```suggestion
const bot = new Bot("");
```

**Comment by KnorpelSenf** [comment]
and many more such cases

**Comment by KnorpelSenf** [comment]
`deno fmt`

File: site/docs/hosting/cloudflare-workers.md [hunk 8]
```diff
@@ -52,9 +52,9 @@ Edit `src/index.js` or `src/index.ts`, and write this code inside:
 // Note that we're importing from 'grammy/web', not 'grammy'.
 import { Bot, webhookCallback } from "grammy/web";
 
-// You can replace `BOT_TOKEN` with your bot token, but it is better to store it in an environment variable.
-// For more on this, see https://developers.cloudflare.com/workers/platform/environment-variables/#secrets-on-deployed-workers.
-// You should also replace `BOT_INFO` with your bot info obtained from `bot.api.getMe()`.
+// The following line of code assumes that you have configured the secrets BOT_TOKEN and BOT_INFO.
```
Comment (KnorpelSenf) [inline site/docs/hosting/cloudflare-workers.md:55]: Should we tell them how to access these values, rather than just using an undeclared variable?

Comment (roj1512) [reply site/docs/hosting/cloudflare-workers.md:55]: It is going to be declared as a global variable in runtime.

Comment (KnorpelSenf) [reply site/docs/hosting/cloudflare-workers.md:55]: Oof.

**Review by KnorpelSenf — APPROVED** [review]
LGTM. Thank you!

**Comment by KnorpelSenf** [comment]
Changes to 67 files and all of them are nitpicks. What a wonderful PQ. The embodiment of what we care about.
